CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -166,15 +171,24 @@ class EditingViewController: UIViewController {
 
     private func setupSections() {
         if isShowingGroups {
-            setupSectionsFomGroups()
+//            setupSectionsFomGroups()
+            let organizationStrings = Constants.Options.organizations.map { $0.name }
+            let yoursAndMore = removeDuplicates(yourStrings: user.groups, moreStrings: organizationStrings)
+            let stringToGroup = { ItemType.group(Constants.Options.organizationsMap[$0]!) }
```

Comment:
do we want to force unwrap here? since its a constant I can see why though..

Reply:
This is just for now! Eventually I want to move all of this to the backend!